subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ReportTestListener.kt

```diff
@@ -12,59 +12,37 @@ class ReportTestListener : RunListener() {
     private val report: Report by lazy { reportInstance }
 
     override fun testStarted(description: Description) {
-        Log.d(TAG, "Receive testStarted event for ${description.displayName}")
-
-        try {
+        processEvent("start", description.displayName) {
             report.startTestCase()
-        } catch (t: Throwable) {
-            Log.e(
-                TAG,
-                "Failed to process testStarted event for ${description.displayName} with error: ${t.message}"
-            )
-            report.sentry.sendException(t)
         }
-
-        Log.d(TAG, "TestStarted event processing completed for ${description.displayName}")
     }
 
     override fun testFinished(description: Description) {
-        Log.d(TAG, "Receive testFinished event for ${description.displayName}")
-
-        try {
+        processEvent("finish", description.displayName) {
             report.reportTestCase()
-        } catch (t: Throwable) {
-            Log.e(
-                TAG,
-                "Failed to send testFinished event for ${description.displayName} with error: ${t.message}"
-            )
-            report.sentry.sendException(t)
         }
-        Log.d(TAG, "TestFinished event processing completed for ${description.displayName}")
```

Comment:
Before we log the event completed even if we had an error while processing. It made harder to realize what actually happen and how